### DISCUSSION TITLE
Recieving prerendering page error since yesterday in my all deployments.

### DISCUSSION THREAD

**Discussion by ItzCrazyKns** [vercel/community#3745] [Help]
### Summary

Hey, I am recieving prerendering page error since yesterday in my all deployments. I haven't made any changes to the code I've only changed the production env. It was working fine before but now its constantly giving that error. I've tried reverting the code back but it is still giving error `
Error occurred prerendering page "/ad". Read more: https://nextjs.org/docs/messages/prerender-error
09:28:06.899 | [object Object]
`

### Steps to Reproduce

- Clone `https://github.com/manifoldmarkets/manifold`,
- Deploy it,
- You will see the error in the deployment logs

**Comment by ItzCrazyKns** [comment]
And its giving the error message as [Object object]

```
Error occurred prerendering page "/ad". Read more: https://nextjs.org/docs/messages/prerender-error
10:24:51.974 | [object Object]
10:24:51.998 |  
10:24:51.999 | Error occurred prerendering page "/latestposts". Read more: https://nextjs.org/docs/messages/prerender-error
10:24:51.999 | [object Object]
10:24:52.021 |  
10:24:52.022 | Error occurred prerendering page "/leaderboards". Read more: https://nextjs.org/docs/messages/prerender-error
10:24:52.022 | [object Object]
10:24:52.400 |  
10:24:52.400 | Error occurred prerendering page "/date-docs". Read more: https://nextjs.org/docs/messages/prerender-error
10:24:52.401 | [object Object]
10:24:52.401 | - info Generating static pages (52/52)
10:24:52.402 |  
10:24:52.402 | > Export encountered errors on following paths:
10:24:52.403 | /ad
10:24:52.403 | /date-docs
10:24:52.403 | /latestposts
10:24:52.403 | /leaderboards
10:24:52.500 | error Command failed with exit code 1.
10:24:52.500 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
10:24:52.525 | Error: Command "yarn run build" exited with 1
```

**Comment by amyegan** [comment]
The https://nextjs.org/docs/messages/prerender-error page recommended in the error has some suggestions:

> - Make sure to move any non-pages out of the pages folder
> - Check for any code that assumes a prop is available, even when it might not be
> - Set default values for all dynamic pages' props (avoid undefined, use null instead so it can be serialized)
> - Check for any out of date modules that you might be relying on
> - Make sure your component handles fallback if it is enabled in getStaticPaths. [Fallback docs](https://nextjs.org/docs/pages/api-reference/functions/get-static-paths#fallback-false)
> - Make sure you are not trying to export (output: 'export' or next export) pages that have server-side rendering enabled [getServerSideProps](https://nextjs.org/docs/pages/building-your-application/data-fetching/get-server-side-props)

**Comment by ItzCrazyKns** [comment]
Okay and what about the error message which is bieng sent in form of a json object which is not bieng stringified?
